backend/simpleLambda/index.js

```diff
@@ -1,51 +1,35 @@
-const messafe='null'
+exports.handler = async (event) => {
+  console.log("event: " + JSON.stringify(event));
+  console.log("----- lambda start");
```

Comment:
This will be running in a serverless environment, so console.log statements won't be available to us

Reply:
Yeah , I am going to clean the code .I just use them when I test the code .Thank you for your tips.